src/apps/sommelier/positions.ts

```diff
@@ -0,0 +1,101 @@
+import { logger } from '../../log'
+import { getClient } from '../../runtime/client'
+import { getTokenId } from '../../runtime/getTokenId'
+import { toDecimalNumber } from '../../types/numbers'
+import {
+  PositionDefinition,
+  PositionsHook,
+  PricePerShareContext,
+  UnknownAppTokenError,
+} from '../../types/positions'
+import { cellarV0821Abi } from './abis/cellar'
+import { getSommStrategiesData } from './sommelierApi'
+
+const hook: PositionsHook = {
+  getInfo() {
+    return {
+      id: 'sommelier',
+      name: 'Sommelier',
+      description: 'Sommelier vaults',
```

Comment:
They now market themselves as just "somm".
```suggestion
      id: 'somm',
      name: 'Somm',
      description: '', // Not sure anywhere
```

Reply:
this is confusing, i see that they are "somm" on their app but the documentation website is still "sommelier" 😅 i can update this whole thing to reference "somm" only to avoid confusion...

Reply:
Yes, this is recent, it seems they'll gradually update everything.